src/mixins/add-edit-org-person-mixin.ts

```diff
@@ -197,22 +196,6 @@ export default class AddEditOrgPersonMixin extends Vue {
     return (!this.showCompletingPartyRole && !this.showIncorporatorRole && this.showDirectorRole)
   }
 
-  /**
-   * Whether to show the delivery address by default.
-   * Applies to some entity types only.
-   */
-  get showDeliveryAddressByDefault (): boolean {
-    return [
-      CorpTypeCd.COOP,
-      CorpTypeCd.BENEFIT_COMPANY,
-      CorpTypeCd.BC_CCC,
-      CorpTypeCd.BC_COMPANY,
-      CorpTypeCd.BC_ULC_COMPANY,
-      CorpTypeCd.SOLE_PROP,
-      CorpTypeCd.PARTNERSHIP
-    ].includes(this.getEntityType)
```

Comment:
I believe this list, plus the 4 continued in types, is a full list of all the entity types that we do anything with, so with this always being True, I removed it from the condition statement on line 242 below.